pilot/pkg/model/proxy_config.go

```diff
@@ -140,6 +141,9 @@ func mergeWithPrecedence(pcs ...*meshconfig.ProxyConfig) *meshconfig.ProxyConfig
 		if pcs[i].GetImage() != nil {
 			merged.Image = pcs[i].GetImage().DeepCopy()
 		}
+		if pcs[i].GetTracing() != nil {
```

Comment:
Here it is to distinguish unset or disabled. We need to discuss how to express that.  Normally, if a pointer == nil, it means unset. If it is specified with empty struct, we can use it as disabled